examples/error-match-statement/error_match_statement.bal

```diff
@@ -21,13 +21,17 @@ public function main() {
 
 function basicMatch(any|error v) {
     match v {
-        var (tVar1, tVar2) => io:println("Matched a value with a tuple shape");
+        var [tVar1, tVar2] => io:println("Matched a value with a tuple shape");
         var { message, fatal } =>
                             io:println("Matched a value with a record shape");
         // If the variable `v` contains an `error` value, it will be matched
-        // to this pattern and the reason string and detail map will be
+        // to this pattern and the reason string and detail record will be
         // destructed within the pattern block.
-        var error(reason, detail) => io:println("Matched an error value : "
-                + io:sprintf("reason: %s, detail: %s", reason, detail));
+        var error(reason, message=message) => io:println("Matched an error value : "
+                + io:sprintf("reason: %s, message: %s", reason, message));
+
+        // If a rest pattern is used the error details will be recorded in a map
```

Comment:
```suggestion
        // If a REST pattern is used, the error details will be recorded in a map.
```

Reply:
Since we are talking about rest functional parameters and not http REST, should we just keep it as is?